website/docs/d/marketplace_image_beta.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_marketplace_image_beta"
+description: |-
+  Gets image ID of an image from its label name.
```

Comment:
Gets local image ID?

Reply:
for now we haven't introduce the concept of local image in terraform. Do we want that?

Reply:
It has been already introduced in issues... The idea is just to be clear on what do we exactly expose with this data source on the description.